models/services.ts

```diff
@@ -4,9 +4,9 @@ export interface Service {
   id: string
   name: string
   description: string
-  address?: string[] // list of IDs linking to locations table
   url: string
   email?: string
+  addresses?: string[] // list of IDs linking to locations table
```

Comment:
Any reason for changing from `address` to `addresses`? It looks like the field is still `address` in the dev Airtable

Reply:
Not really lol, I thought about editing some of the other field names too. Fo example some are capitalized and some are not. What do you think? Should I just change it back here, or is it easy enough to just update the dev & prod airtable

Reply:
Eh I guess I should not mess with the actual services table. ServiceLists i don't mind messing with. Will just change it back

Reply:
Reverted